docs/sources/enterprise/access-control/assign-rbac-roles.md

```diff
@@ -151,139 +123,69 @@ roles:
-
-1. Open the YAML configuration file and locate the `builtInRoles` section.
-
-1. Refer to the following table to add attributes and values.
-
-   | Attribute      | Description                                                                                                                                                                                                                                                  |
-   | -------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-   | `name`         | Enter the name of the custom role.                                                                                                                                                                                                                           |
-   | `version`      | Enter the custom role version number. Assignments are updated if the version of the role is greater than or equal to the version number stored internally. If you are updating a role assignment, you are not required to increment the role version number. |
-   | `orgId`        | If you do not enter an `orgId`, it inherits the `orgId` from `role`. For global roles the default `orgId` is used. `orgId` in the `role` and in the assignment must be the same for non-global roles.                                                        |
-   | `permissions`  | Enter the permissions `action` and `scope` values. For more information about permissions actions and scopes, refer to [LINK]                                                                                                                                |
-   | `builtInRoles` | Enter the `name` of an organization role, for example `Viewer`, `Editor`, or `Admin`, or enter `Grafana Admin`.                                                                                                                                              |
+**Note**: The roles don't have to be defined in the provisioning configuration files to be assigned. As long as they exist in the database, they are assignable.
 
-1. Reload the provisioning configuration file.
+**Remove a role assignment from a team:**
 
-   For more information about reloading the provisioning configuration at runtime, refer to [Reload provisioning configurations]({{< relref "../../http_api/admin/#reload-provisioning-configurations" >}}).
+If you want to remove an assignment from a team, add `state: absent` to the `teams > roles` section, and reload the configuration file.
 
-The following example assigns the `users:editor` custom role to the basic editor and admin roles.
+The following example creates the `custom:users:writer` role and assigns it to the `user admins` team along with the `fixed:users:writer` role, but ensures the `user writers` team does not have them:
```

Comment:
I'm not sure what "but ensures the user writers team does not have them" means. Doesn't have what?

Reply:
Maybe not the best phrasing I agree :thinking: 
Provisioning will remove `custom:users:writer` and `fixed:users:writer` assignments to `user writers` if such assignments exist.

Reply:
Gotcha, ok - I think that sentence is trying to achieve quite a bit. Maybe break it up, something like:

The following example:
-  Creates the `custom:users:writer` role
- Assigns the `custom:users:writer` role and the `fixed:users:writer` role to the `user admin` team
- Removes the `custom:users:writer` and the `fixed:users:writer` assignments from the `user writers` team, if those assignments exist.